### ISSUE TITLE
Can you add BillingGooglePlay in your Godot Project? 

### ISSUE THREAD

**Issue by al-bezd** [chazzledazzle10/Hello-world#2]
Hello i have some problem, and i want know, you win this problem?
https://github.com/godotengine/godot-google-play-billing/issues/35

**Comment by chazzledazzle10** [comment]
You need to make sure everything is updated on the phone ie newest version of android installed. I might be able to help you more if you provide more details.

**Comment by al-bezd** [comment]
> You need to make sure everything is updated on the phone ie newest version of android installed.

Yes, I did it in emulator, and this was not worked, but i try it in my phisycal phone and it worked now, thanks you for answer, good luck
